src/main/webapp/js/finscholar/homepagecontroller/homepagecontroller.js

```diff
@@ -60,6 +61,10 @@ class HomePageController extends PageController {
                                CollegeListView, 
                                 ScholarshipListView, 
                                ];
+    /** @private @const {!function(!jsaction.ActionFlow): Promise<undefined>} */
+    this.bindedListItemOnlickHandler_ = this.handleListItemOnclickEvent_.bind(this);
+    /** @private {CommonListView} Registers the current list (if any) opened. */
+    this.currentList_ = null;
```

Comment:
@hzimz I thought about changing this to onFocus but later I found my onscoll event manipulates instance variables in listHandlers, which are created after one navbar is clicked...